### PR TITLE
Release/v0.28.0

### PR DESCRIPTION
**Pull request by ebuchman**
[Changelog](https://github.com/tendermint/tendermint/blob/239ebe20769b79f46a2ddca88de78e8044543e70/CHANGELOG.md#v0280)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
+*January 16th, 2019*
+
+Special thanks to external contributors on this release:
+@fmauricios, @gianfelipe93, @husio, @needkane, @srmo, @yutianwu
+
+This release is primarily about upgrades to the `privval` system -
+separating the `priv_validator.json` into distinct config and data files, and
+refactoring the socket validator to support reconnections.
+
+XXX: Please backup your existing `priv_validator.json` before using this
```
Comment (Liamsi) [inline CHANGELOG.md]:
```suggestion
**Note:** Please backup your existing `priv_validator.json` before using this
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,60 @@
...
+
+### BUG FIXES:
+
+- [build] [\#3085](https://github.com/tendermint/tendermint/issues/3085) Fix `Version` field in build scripts (@husio)
+- [crypto/multisig] [\#3102](https://github.com/tendermint/tendermint/issues/3102) Fix multisig keys address length
+- [crypto/encoding] [\#3101](https://github.com/tendermint/tendermint/issues/3101) Fix `PubKeyMultisigThreshold` unmarshalling into `crypto.PubKey` interface
+- [p2p/conn] [\#3111](https://github.com/tendermint/tendermint/issues/3111) Make SecretConnection thread safe
+- [rpc] [\#3053](https://github.com/tendermint/tendermint/issues/3053) Fix internal error in `/tx_search` when results are empty
+  (@gianfelipe93)
+- [types] [\#2926](https://github.com/tendermint/tendermint/issues/2926) Do not panic if retrieving the private validator's public key fails
```
Comment (Liamsi) [inline CHANGELOG.md]: Nit: I probably wrote this but "retrieving the private validator's public key" reads a bit weirdly. Change to privval or remote signer?

```suggestion
- [types] [\#2926](https://github.com/tendermint/tendermint/issues/2926) Do not panic if retrieving the remote signer's public key fails
```

File: README.md [hunk 3]
```diff
@@ -145,8 +122,40 @@ data into the new chain.
...
+
+* [Amino](http://github.com/tendermint/go-amino), reflection-based proto3, with
+  interfaces
+* [IAVL](http://github.com/tendermint/iavl), Merkleized IAVL+ Tree implementation
+
+### Applications
+
+* [Cosmos SDK](http://github.com/cosmos/cosmos-sdk); a cryptocurrency application framework
+* [Ethermint](http://github.com/cosmos/ethermint); Ethereum on Tendermint
+* [Many more](https://tendermint.com/ecosystem)
```
Comment (Liamsi) [inline README.md]: Do we want to link the community maintained list here, too?  
https://forum.cosmos.network/t/list-of-projects-in-cosmos-tendermint-ecosystem/243

File: docs/spec/consensus/signing.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+The structure of a vote for signing looks like:
+
+```
+type CanonicalVote struct {
+	Type      SignedMsgType // type alias for byte
+	Height    int64         `binary:"fixed64"`
+	Round     int64         `binary:"fixed64"`
+	Timestamp time.Time
+	BlockID   BlockID
```
Comment (Liamsi) [inline docs/spec/consensus/signing.md]: Not sure how this happened (again). The order should be:
```
BlockID   BlockID
Timestamp time.Time
```
It's correct in the go-code: 
https://github.com/tendermint/tendermint/blob/dcb8f885256e40a088e0901ee127fe095647cea0/types/canonical.go#L39-L40
https://github.com/tendermint/tendermint/blob/dcb8f885256e40a088e0901ee127fe095647cea0/types/canonical.go#L30-L31
https://github.com/tendermint/tendermint/blob/dcb8f885256e40a088e0901ee127fe095647cea0/types/proposal.go#L28-L29
https://github.com/tendermint/tendermint/blob/dcb8f885256e40a088e0901ee127fe095647cea0/types/vote.go#L55-L56

File: README.md (hunk 3 above)
Comment (zramsay) [reply README.md]: hmm, we should update the ecosystem.json - that URL is generated from: https://github.com/tendermint/tendermint/blob/master/docs/app-dev/ecosystem.json

**Pull request merged**